substrate-node/support/src/traits.rs

```diff
@@ -4,7 +4,10 @@ pub trait Tfgrid<AccountId, Name, PublicIP> {
     fn is_twin_owner(twin_id: u32, who: AccountId) -> bool;
 }
 
-pub trait ChangeNode<PubConfig, If> {
-    fn node_changed(node: Option<&super::types::Node<PubConfig, If>>, new_node: &super::types::Node<PubConfig, If>);
-    fn node_deleted(node: &super::types::Node<PubConfig, If>);
-}
\ No newline at end of file
+pub trait ChangeNode<CityName, CountryName, Location, PubConfig, If> {
```

Comment:
see if you can create type alias to improve readability

Reply:
Tried a while creating type alias for trait definition without success...
Any suggestion is welcome if you see a way to do it